runner/src/cfc.ts

```diff
@@ -0,0 +1,156 @@
+// This is a simple form of the policy, where you express the basics of the partial ordering.
+
+import { type JSONSchema } from "@commontools/builder";
+
+// We'll work with the transitive closure of this graph.
+const defaultClassificationPolicy = new Map<string, string[]>([
```

Comment:
that's technically the principals lattice, not yet the full policy. or more precisely in the full system, this will capture the user's configuration of trust delegations, where as the policy for what can and can't be done to data is in the confidentiality label attached to the data. (let's discuss this on Monday for sure)